### ISSUE TITLE
Split the ggplot-related functions into a separate package?

### ISSUE THREAD

**Issue by gavinsimpson** [eliocamp/metR#95]
The ggplot-based functions in *metR* are really very useful, and not just for *metR*, but for many other packages. However, adding a dependency on *metR* adds a long chain of other dependencies, many of which have nothing to do with the ggplot parts. This makes me somewhat reticent to base packages I write *metR* just to use a single geom (in my case I'd really like to use the contour labelling geoms).

Would you ever consider unbundling the ggplot-based parts of *metR* into separate package(s) that *metR* would import from and thus allow other package developers to import from also?

**Comment by eliocamp** [comment]
Yes. metR grew more or less organically from my needs as a researcher so it's somewhat eclectic. Exploding it into it's components could be a good idea. Another option is to move more dependencies to Suggests and add the relevant checks. 

I'll consider it. But being a rather involved process, I'm afraid it won't be quick :sweat_smile:
